secator/hooks/json.py

```
@@ -0,0 +1,140 @@
+	for output_type in FINDING_TYPES:
+		type_name = output_type.get_name()
+		sidecar_path = _get_sidecar_path(self, type_name)
+		items = []
+		if sidecar_path.exists():
+			try:
+				with open(sidecar_path, 'r') as f:
+					for line in f:
+						line = line.strip()
+						if line:
+							items.append(json.loads(line))
+			except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)